### ISSUE TITLE
desktop version is not working for ubuntu 16.04 LTS 

### ISSUE THREAD

**Issue by learncodingforweb** [jpnurmi/flutter_libserialport#16]
```
/home/sukhdeep/project/git/bt/ft/webdev/nodejs/multiplatform/ft/menuplator/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/linux/flutter_libserialport_plugin.cc:40:3: error: use of undeclared identifier 'setenv'; did you mean 'g_setenv'?
```

```
name: learn
description: A new Flutter project.
publish_to: 'none' # Remove this line if you wish to publish to pub.dev
version: 1.0.0+1

environment:
  sdk: ">=2.7.0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter

  cupertino_icons: ^1.0.2
  flutter_libserialport: ^0.2.0

dev_dependencies:
  flutter_test:
    sdk: flutter

flutter:
  uses-material-design: true

```

**Comment by jpnurmi** [comment]
Hi, I have published 0.2.1 with the fix proposed in the error message. Hopefully that helps.

**Comment by learncodingforweb** [comment]
problem is resolved in 0.2.1
